NimbleMedium/Sources/Injection/Resolver+Injection.swift

```diff
@@ -156,18 +156,20 @@ extension Resolver: ResolverRegistering {
             ArticleRowViewModel(article: args.get())
         }.implements(ArticleRowViewModelProtocol.self)
         register { CreateArticleViewModel() }.implements(CreateArticleViewModelProtocol.self)
+        register(EditArticleViewModelProtocol.self) { _, args in
+            EditArticleViewModel(slug: args.get())
+        }.scope(.shared)
         register { EditProfileViewModel() }.implements(EditProfileViewModelProtocol.self)
         register { FeedsViewModel() }.implements(FeedsViewModelProtocol.self).scope(.cached)
         register(FeedsTabViewModelProtocol.self) { _, args in
             FeedsTabViewModel(tabType: args.get())
-        }
-        .scope(.unique)
+        }.scope(.unique)
         register { HomeViewModel() }.implements(HomeViewModelProtocol.self).scope(.cached)
-        register { LoginViewModel() }.implements(LoginViewModelProtocol.self).scope(.cached)
-        register { SideMenuActionsViewModel() }.implements(SideMenuActionsViewModelProtocol.self).scope(.cached)
-        register { SideMenuHeaderViewModel() }.implements(SideMenuHeaderViewModelProtocol.self).scope(.cached)
-        register { SideMenuViewModel() }.implements(SideMenuViewModelProtocol.self).scope(.cached)
-        register { SignupViewModel() }.implements(SignupViewModelProtocol.self).scope(.cached)
+        register { LoginViewModel() }.implements(LoginViewModelProtocol.self).scope(.shared)
```

Comment:
Should we update with this: 
```suggestion
        register(LoginViewModelProtocol.self) { LoginViewModel() }.scope(.shared)
```

Since `implements` is for applying multiple protocols

Reply:
@suho yes, we have another task for it #237 here. This is out of scope for this ticket 😁